src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const maxDiscount = 50;
+  const minDiscount = 20;
+  const rentCost = 40;
+  let dirtyCost = days * rentCost;
+
+  if (days >= 3 && days < 7) {
+    dirtyCost = dirtyCost - minDiscount;
+  } else if (days >= 7) {
```

Comment:
would be be great to pass days numbers into variables too
and avoid using `else if` - it make your code more complicated for reading
you can use 2 if's (>= 3) and (>=7)